internal/cli/usercmd/service.go

```diff
@@ -244,10 +244,12 @@ func (c *EpinioClient) ServiceList() error {
 
 	msg := c.ui.Success().WithTable("Name", "Created", "Catalog Service", "Status")
 	for _, service := range resp.Services {
-		msg = msg.WithTableRow(service.Meta.Name,
-			fmt.Sprintf("%v", service.Meta.CreatedAt),
+		msg = msg.WithTableRow(
+			service.Meta.Name,
+			service.Meta.CreatedAt.String(),
```

Comment:
Oh! I will have to make a PR for all the other places showing stamps. I used `fmt.Sprintf("%v", ...)`. This is much simpler.